constitution.tex

```diff
@@ -794,6 +794,9 @@
 	\item The candidates will be notified of their nomination.
 		Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 		A list of all nominees who have accepted their nominations will be posted shortly thereafter.
+	\item The date and time of the candidate(s)'s speeches will be announced by a member of the Executive Board at least five (5) days prior to the event. 
+		All Active Members are expected to attend this Election, as specified in \ref{Expectations of House Members}.
+		Exemptions to attendance may be given at the discretion of the Evaluations Director.
```

Comment:
`Exemption` isn't well defined. `Exempt` as in "they qualify equivalently as having attended"? Or `Exempt` in the manner of house meeting excusals, which are simply "they were absent, but they can write down an excuse"?

Reply:
Also nit: `s/given/granted`